internal/pkg/cli/svc_logs.go

```diff
@@ -84,6 +90,10 @@ func newSvcLogOpts(vars wkldLogsVars) (*svcLogsOpts, error) {
 			configStore: configStore,
 			deployStore: deployStore,
 			sel:         selector.NewDeploySelect(prompt.New(), configStore, deployStore),
+			newSvcDescriber: func(s *session.Session) serviceDescriber {
+				return ecs.New(s)
+			},
+			sessProvider: sessProvider,
```

Comment:
To make the unit tests not depend on the `~/.aws/config` file and the `sessProvider`, we can slightly alter this:
```go
opts.newSvcDescriber = func(envCfg *config.Environment) serviceDescriber {
	sess, err := sessProvider.FromRole(envCfg.ManagerRoleARN, envCfg.Region)
	if err != nil {
		return fmt.Errorf("get session %s: %w", o.envName, err)
	}
	return ecs.New(sess)
}
```

Reply:
I also have an alternative. So what we can do is set `newSvcDescriber` shown in the below snipped, inside [this](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/svc_logs.go#L89-L116) block on L102 so that we can reutilize the sess we created for `NewWorkloadClient`. 

```
opts.newSvcDescriber = func() serviceDescriber {
			return ecs.New(sess)
		}
```

What do you think about that?